src/components/BulkEnrollmentPage/stepper/ReviewList.test.jsx

```diff
@@ -53,6 +54,10 @@ describe('ReviewList', () => {
     render(<ReviewList {...defaultProps} />);
     expect(screen.getByText(defaultProps.subject.title)).toBeInTheDocument();
   });
+  it('renders delete button hover title', () => {
+    render(<ReviewList {...defaultProps} />);
+    expect(screen.getAllByTitle(defaultProps.subject.removal)[0]).toBeInTheDocument();
```

Comment:
as a quick a11y coverage wondering if also getting the button by aria-label if possible also is a good idea? and even by title?

Reply:
scratch the 'title' part I see you already do it

Reply:
Not sure if I'm misunderstanding, but it sounds like you're asking for a test for the delete button's title and aria text, which:
A. this test you're commenting on is for the title (which is why I'm wondering if I'm misunderstanding)
B. I mention in the PR description that react testing library doesn't really offer a way to actually test that the aria label appears to the user. I can maybe do a unit test, but that seems a bit redundant since 1. the mechanism that is passing in this text is being tested already by getting the title text here and 2. if we're just testing that alt renders as aria-label in the document, we're just testing the IconButton component's functionality, which is already done in the IconButton's own test in Paragon.

Reply:
Yeah I agree. I think testing by title covers our based for a11y for the title you just added. I looked at https://testing-library.com/docs/queries/bylabeltext for clues.. it seems aria-label may be covered by ByLabelText but I would not worry about it in this codebase since it's part of the IconButton features.

I was thinking if there a way for us to capture that we are passing the correct 'alt' value as well

Reply:
Basically a test case for (somehow) `Update existing alt= field on IconButton to use the altText prop.`

Reply:
I see. This proposed test would cover a case where someone changes the name of the variable and misses updating it on the alt field. Doing so doesn't cause any error, just a prop warning currently for the IconButton component (and we all ignore those). I find the likelihood of such a change very very small, but I can test for it with not too much additional weight besides having to use enzyme again and isn't quite the same as what the IconButton test in Paragon is testing, so it's not actually redundant. I will commit that in a sec.

I just very much want to avoid redundant tests. Eventually all the tests we add will actually impact build performance as well as container size. Each additional test on its own is of course very very minor, but it eventually adds up. I've been in that environment, so I don't want to go back to that state of things lol.